guides/common/assembly_managing-alternate-content-sources.adoc

```diff
@@ -1,3 +1,7 @@
 include::modules/con_managing-alternate-content-sources.adoc[]
 
-include::modules/proc_configuring-alternate-content-sources.adoc[leveloffset=+1]
+include::modules/proc_configuring-custom-alternate-content-sources.adoc[leveloffset=+1]
+
+include::modules/proc_configuring-simplified-alternate-content-sources.adoc[leveloffset=+1]
+
+include::modules/proc_configuring-rhui-alternate-content-sources.adoc[leveloffset=+1]
```

Comment:
@maximiliankolb should there be an `ifndef::orcharhino[]` here?

Reply:
No. My understanding is that this could potentially be used to sync RHEL packages into orcharhino if orcharhino runs in the cloud.